db/migrate/20180724140319_remove_change_note.rb

```diff
@@ -0,0 +1,17 @@
+# Remove a public change note (dated 20 July 2018) for "VAT Notice 708: buildings and construction"
+# Ticket: https://govuk.zendesk.com/agent/tickets/2927623
+# Page: https://www.gov.uk/government/publications/vat-notice-708-buildings-and-construction
+#
+# Prior steps:
+# Queried `Document` for the `content_id`, then grabbed the `id` of the `ChangeNote` from the result
+
+class RemoveChangeNote < ActiveRecord::Migration[5.1]
+  def up
+    # Get rid of change note
+    ChangeNote.delete(157789)
```

Comment:
Once you've got the `ChangeNote`, you will need to do [something like this](https://github.com/alphagov/publishing-api/blob/8c43bf55d0c58b3da9650560c882f3819c79fcff/db/migrate/20180215112954_fix_whitehall_remove_change_note.rb#L18-L20) to represent the content item to the content store. Similarly, you'll need to include `disable_ddl_transaction!` at the top for that to work.

I also wonder, for such a simple change, it might be easier to just run the command directly on the machine rather than creating a migration.

Reply:
> I also wonder, for such a simple change, it might be easier to just run the command directly on the machine rather than creating a migration.

I think so, but I also like the idea of keeping some kind of record of this change, so I like having it as a migration.

Reply:
@thomasleese added `disable_ddl_transaction!` and the `RepresentDownstream` for production.

The PR fails and asks for a `rails db:migrate` which I can't successfully run as I don't have the DB setup yet 😞.

Reply:
If you wanted to skip that, since you don't make any schema changes, you could [manually update the date in the schema file](https://github.com/alphagov/publishing-api/blob/master/db/schema.rb#L13) to the date of your migration.